woocommerce/woocommerce
# add/a2p api-core-tests product properties tests

### All Submissions:






### Changes proposed in this Pull Request:


Adding CRUD tests for the tax rates and tax classes to ensure we have better test coverage
Closes 366-gh-woocommerce/woocommerce-quality and 363-gh-woocommerce/woocommerce-quality.

### How to test the changes in this Pull Request:

1. Pull down the code
2. Navigate to woocommerce/plugins/woocommerce
3. Execute tests with USE_WP_ENV=1 pnpm playwright test --config=tests/api-core-tests/playwright.config.js tests/api-core-tests/tests/products/products-crud.test.js
4. The products tests should execute and pass successfully including the new tags, attributes and attributes terms tests

### Other information:




### FOR PR REVIEWER ONLY:
